ports/stm32f4/common-hal/pulseio/PulseIn.c

```diff
@@ -0,0 +1,79 @@
+/*
+ * This file is part of the MicroPython project, http://micropython.org/
+ *
+ * The MIT License (MIT)
+ *
+ * Copyright (c) 2017 Scott Shawcroft for Adafruit Industries
```

Comment:
change to you

Reply:
This is marked resolved but it doesn't appear updated.